modules/apps/frontend-js/frontend-js-web/test/liferay/util/portlet_url/create_url.es.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+'use strict';
+
+import createURL from '../../../../src/main/resources/META-INF/resources/liferay/util/portlet_url/create_url.es';
+
+describe('Liferay.Util.PortletURL.createURL', () => {
+	beforeEach(() => {
```

Comment:
If there is `beforeEach`, there needs to be an `afterEach`, that resets everything set in `beforeEach`